classes/Database/Install.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Database;
+
+class Install
+{
+    /**
+     * @var \Ps_Googleanalytics
+     */
+    private $module;
+
+    public function __construct(\Ps_Googleanalytics $module)
+    {
+        if (\Shop::isFeatureActive()) {
```

Comment:
You should use namespaces for better readability instead (everywhere in the codebase)
```suggestion
        if (Shop::isFeatureActive()) {
```

Reply:
As the module must work on PrestaShop 1.6.0.0 it seems it's not possible.
`Shop` doesn't have a namespace on 1.6.  If you have an idea I'm interested

Reply:
`use Shop;` is enough, even for the 1.6, it's working for PHP>=5.3